apps/prairielearn/src/pages/userSettings/userSettings.html.ts

```diff
@@ -115,9 +120,27 @@ export function UserSettings({
                         Try a new navigation experience for instructors that makes accessing your
                         course simpler, faster, and more intuitive.
                       </div>
+                      <div class="small text-muted">
+                        Have feedback? <a href="${discussionUrl}" target="_blank">Share it here!</a>
+                      </div>
```

Comment:
Nit: for accessibility reasons, it's best to have links be self-descriptive. "Share it here" isn't understandable in isolation. The overall rewording was an attempt to resolve this.

```suggestion
                        course simpler, faster, and more intuitive. <a href="${discussionUrl}" target="_blank">Share your feedback</a> to help us improve the new design.
                      </div>
```